.github/workflows/ci.yml

```diff
@@ -12,6 +12,7 @@ jobs:
           - windows-latest
         node:
           - 10
```

Comment:
Can Node 10 testing be dropped?

Reply:
Since Node 10 is no longer supported https://github.com/nodejs/Release#readme
and mdx 2 is a major release.
I'd lean yes, drop 10 and add 16

Reply:
same